Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -482,6 +482,19 @@ open class ChatMessageListVC:
                 "Tapped an avatarView. To customize the behavior, override messageContentViewDidTapOnAvatarView. Path: \(indexPath)"
             )
     }
+    
+    /// This method is triggered when delivery status indicator on the message at the given index path is tapped.
+    /// - Parameter indexPath: The index path of the message cell.
+    open func messageContentViewDidTapOnDeliveryStatusIndicator(_ indexPath: IndexPath?) {
+        guard let indexPath = indexPath else { return log.error("IndexPath is not available") }
```

Comment:
@evsaev is this action going to be piped to one of the delegates?

Reply:
This method is triggered when the delivery indicator is tapped and can be overridden in `ChatMessageListVC` subclass if needed. The same is done for all other `ChatMessageContentViewDelegate`'s methods